app/views/resources/index.scala.html

```diff
@@ -130,10 +131,26 @@
             }
           }
 
+          @if(Feature.intakeSupported) {
```

Comment:
I would like to perhaps investigate only showing this button if `controllers.Resources.intake` is enabled for the user? @discordianfish is that possible to test?